libvirt/disk_def.go

```diff
@@ -50,3 +53,13 @@ func newCDROM() defDisk {
 
 	return disk
 }
+
+func generateWWN(OUI string, strlen int) string {
```

Comment:
Could it be possible to name this function `randomWWN`? We have a couple of `random*` functions so this woiuuld follow the same pattern...